src/any-type-field/any-type-field.component.html

```diff
@@ -1,23 +1,23 @@
-<div [ngSwitch]="componentType">
-  <div *ngSwitchCase="'table-list'">
+<div [ngSwitch]="componentType" [ngClass]="{disableCursorClass: schema && schema['x_editor_disabled'] ? true : false}">
```

Comment:
@harunurhan The `disableCursorClass` set cursor to not-allowed. The `disableEventsClass` is actually disabling the pointer-events. I have it both because if apply the both classes to the same div the cursor is the default one because there is a known conflict when applying pointer-events:none and cursor:not-allowed. The solution was to apply it to nested divs. If we don't want the cursor thing I can apply only the pointer-events:none css to the div you mentioned.